src/dynamics/video_player/player/player.html

```diff
@@ -36,9 +36,10 @@
             ba-gallerysidebar="{{gallerysidebar}}"
         ></ba-{{dynnext}}>
         <div class="{{css}}-tooltips-list-container"
-             ba-repeat="{{tooltip :: tooltips}}"
+            ba-repeat="{{tooltip :: tooltips}}"
         >
             <ba-{{dyntooltip}}
+                id="{{tooltip.id}}"
```

Comment:
Hi @Josewd does it works as expected? 
Seems it's using like element attribute ID not id variable. To pass as component variable here should be `ba-id`

Reply:
Just like that it worked @sambua

Reply:
Great @Josewd , thanks. 
But I think anyway will be better to replace it as `ba-id` as we're not using id attribute in betajs project not cause any conflict on user side projects.

Reply:
with ba-id it did not work, any specific way to select it on css? @sambua

Reply:
I will change it for class selector like this: `class="{{csscommon}}-{{tooltip.id}}"`

Reply:
weird, I've checked with other component and seems passing ba-id and getting it with this.get(`id`); inside component and it was worked as expected.

Reply:
Oh I was trying to getting it on CSS, not JavaScript. That's why